app.py

```diff
@@ -397,6 +402,8 @@ def operator():
 @app.route("/operator/form")
 def operator_form():
     data = t_db.search(Query().Type == "rootchain")
+    data.append(config['NETWORKS']['MAINNET'])
```

Comment:
It look like some `config.ini` field is changed. Where is README.md file which explains this field?

Reply:
It was my mistake. I'm considering how to insert Mainnet and Rinkeby endpoint data to database.